src/core/lib/iomgr/ev_epollex_linux.cc

```diff
@@ -584,30 +585,26 @@ static grpc_error* pollable_create(pollable_type type, pollable** p) {
 
 #ifdef NDEBUG
 static pollable* pollable_ref(pollable* p) {
+  const char* reason = nullptr;
+  grpc_core::DebugLocation debug_location;
 #else
 static pollable* pollable_ref(pollable* p, int line, const char* reason) {
```

Comment:
I would capture `grpc_core::DebugLocation` here and use `DEBUG_LOCATION` when calling `pollable_ref`.

Reply:
done